mapswipe_workers/tests/integration/test_create_footprint_project.py

```diff
@@ -23,8 +23,9 @@ def tearDown(self):
 
     def test_create_footprint_project(self):
```

Comment:
here we should now probably also check if the tasks contain the attributes we would like to have?

not sure what is the best option. maybe you add a new test function and just check the tasks?

Reply:
I added one test which counts how many usernames made it into project_specifics->properties->username. If above 0 made it we can guess that ohsomeinfos and changesets arrived at the db. 

Do you think that is enough or do you have more ideas for tests?

Reply:
perfect, yes I think this should be enough.